scripts/docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: "3"
+
+services:
+
+  # Database
+  couchdb:
+    image: couchdb:2.3
+    restart: unless-stopped
+    env_file: .env
+    volumes:
+      - ./volumes/couchdb:/opt/couchdb/data
+
+  # Cache
+  redis:
+    image: redis:alpine
+    restart: unless-stopped
+    volumes:
+      - ./volumes/redis/data:/data
```

Comment:
I think it should be better to avoid redis with docker/docker-compose: it is not useful as we run only one instance of cozy-stack and it adds some noise.

Reply:
Ok, removed